src/Application/Products/Commands/AddProduct/ProductsMapper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Application.Products.DTOs;
+
+namespace Application.Products.Commands.AddProduct;
+
+internal static class ProductsMapper
+{
+    public static Domain.Entities.Product.Product MapToEntity(this AddProductCommand dto)
```

Comment:
Extension method for a mapper could be a bad choice for the long term, because we might start requiring:
- injected services,
- mocking and testing.

Reply:
It is also a bad choice in a sense that we now have 4 different `ProductMapper.cs` files - confusion might arise as the solution grows.

Reply:
What do you think would be the best alternative?

Reply:
Agree, this should change, I have two ideas:

1. A factory
2. An explicit (or implicit) conversion

Reply:
On second thought, it's okay to have it like that. I'll not be blocking this PR with that petty comment :)